Admin/PageAdmin.php

```diff
@@ -336,7 +336,7 @@ protected function configureFormFields(FormMapper $formMapper)
         if ($this->hasSubject() && !$this->getSubject()->getId()) {
             $formMapper
                 ->with('form_page.group_main_label')
-                    ->add('site', null, array('required' => true, 'read_only' => true))
+                    ->add('site', null, array('required' => true, 'attr' => array('readonly' => 'readonly'))
```

Comment:
This will work with old symfony versions?

Reply:
i didn't check that. I thougth "3.x" means version for Symfony 3. I have tried to use Page bundle with Symfony 3.2 and get error when open  page to create new record.

Reply:
According to the [composer](https://github.com/sonata-project/SonataPageBundle/blob/3.x/composer.json#L33-L45) file, we still support symfony 2.3 on the current stable branch.